src/store/index.js

```diff
@@ -11,15 +11,14 @@ import createSagaMiddleware from 'redux-saga';
 import rootReducer from '../reducers/index';
 import rootSagas from '../sagas';
 
-
-const saga = createSagaMiddleware();
-const middlewares = [saga, thunk];
+export const sagaMiddleware = createSagaMiddleware();
```

Comment:
question: Why do we need to export the saga middleware?

Reply:
We don't need to since we're not testing the sagas, but it will be useful when we start doing so